vars/RedHat.yml

```diff
@@ -28,3 +28,6 @@ __php_opcache_conf_filename: 10-opcache.ini
 __php_fpm_daemon: php-fpm
 __php_fpm_conf_path: "/etc/fpm"
 __php_fpm_pool_conf_path: "/etc/php-fpm.d/www.conf"
+
+__php_fpm_pool_user: apache
```

Comment:
@mamchenkov :
* Session directory : I'm currently testing a PHP app with this configuration, we'll see how it goes.
* Variable for the group : already handled in the PR :)